internal/payloads/pubkey_payload.go

```diff
@@ -8,32 +8,54 @@ import (
 	"github.com/go-chi/render"
 )
 
-type PubkeyPayload struct {
-	*models.Pubkey
+// See models.Pubkey
+type PubkeyRequest struct {
+	Name string `json:"name" yaml:"name"`
+	Body string `json:"body" yaml:"body"`
+}
```

Comment:
Again, out of curiosity, why is that way better?

Reply:
This was alway the idea, in most typical HTTP apps we deal with two kind of types:

* Payloads represent the data that go in (requests) and out (responses). There are very different depending on the endpoint purpose, e.g. "create" endpoint may return nothing, the created data or just ID of the database index. Depends on what you want.
* Models represent tables in SQL, they need to be ideally 1:1 mapping to the fields.

Due to time constraints, we used models for all request/response payloads, but this created confusion - for example it generated an OpenAPI "create pubkey" which requires all fields:

* name
* body
* type
* fingerprint
* fingerprint_legacy

This was confusing, this is actually the purpose of this PR - to split payloads and models into two distinct objects. Now we can create PubkeyRequest that only have relevant fields - name and body. Everything else is calculated by the server and is "read only".